compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.td

```diff
@@ -26,6 +26,13 @@ class FLOW_PureOp<string mnemonic, list<Trait> traits = []> :
 // Partitioned regions
 //===----------------------------------------------------------------------===//
 
+def OpGroupPartitionedRegionOps : OpDocGroup {
+  let summary = "Partitioned region ops";
+  let description = ".";
```

Comment:
Period on purpose?

Reply:
Oops, copy-paste error (had that in my template so I'd remember to end sentences with ".", but I ended up omitting descriptions and just using empty string for many of the groups).